crash_similarity.py

```diff
@@ -15,10 +15,21 @@
 import download_data
 import utils
 
+import datetime
 import pyximport
 pyximport.install()
 
 
+# Checks if the model has been trained in the last 24 hours (using datetime.timedelta class)
```

Comment:
Let's remove "(using datetime.timedelta class)" from the comment, as it is an implementation detail.